source/features/sticky-comment-header.css

```diff
@@ -0,0 +1,55 @@
+html:not([rgh-OFF-sticky-comment-header]) {
+	/* Issue body header */
+	[class^="IssueBodyHeader-module__IssueBodyHeaderContainer"],
+	/* Issue comment header */
+		[data-testid='comment-header'],
+	/* PR body/comment header */
+		.timeline-comment-header {
+		position: sticky;
+		top: calc(var(--base-sticky-header-height, 0px) + 56px);
```

Comment:
What's base-sticky-header-height? I don't see it used anywhere in the repo.


PS: there are a number of variables defined in `sticky-sidebar` so I'd hope to share the same logic for both, maybe moving them to `refined-github.css` if they're really shared. I don't know what's _required_ for this feature to work though.